ntc_templates/templates/cisco_ios_show_lldp_neighbors_detail.textfsm

```diff
@@ -63,6 +70,15 @@ IgnoreDescription
   ^\s*$$
   ^.* -> Error
 
+PowerMDI
+  ^\s+Power\s+Pair:\s+${POWER_PAIR}
```

Comment:
As the variable match is at the end of the line, probably better to follow them with 
`\s*$$` to ensure no capture of spaces

Reply:
Thanks @diepes I've pushed your suggested changes now.